aiokafka/abc.py

```diff
@@ -87,6 +90,17 @@ def on_partitions_assigned(self, assigned):
         pass
 
 
+class AbstractTokenProvider(BaseAbstractTokenProvider):
+    async def token(self):
+        return await asyncio.get_running_loop().run_in_executor(
+            None, self._token)
+
+    @abc.abstractmethod
+    def _token(self):
+        pass
+
```

Comment:
Please also add an `extensions` method, but leave it returning an empty sequence (not abstract method). This way the user would be able to understand how he can add extensions, but will not be required to do so.